### PR TITLE
Add tests for addSnapshotToPolygons function.

### PR DESCRIPTION
**Pull request by ancamaria99**

### CODE REVIEW

File: step-release-vis/src/app/services/environment_test.ts [hunk 1]
```diff
@@ -210,6 +210,108 @@ describe('EnvironmentService', () => {
     expect(result.snapshot).toEqual([]);
   });
 
+  it('#addSnapshotToPolygons snapshot with the 2 same existing candidates', () => {
```
Comment (stofee) [inline step-release-vis/src/app/services/environment_test.ts:213]: maybe add more describe blocks to structure the testes a bit more? (one for all of these would be nice)

Comment (ancamaria99) [reply step-release-vis/src/app/services/environment_test.ts:213]: do you mean to have a 'describe' for each set of tests for a function?

**Comment by stofee** [comment]
yes pls, I was just not about ask it in this pr, so for now, add 1 for the
new tests, and send an other pr with the grouping for the rest ;)
--
Stofee


•  Kristóf Lackner

•  Software Engineer, Tools and Infrastructure

•  stofee@google.com

•  +41794802969
Google Switzerland GmbH, Brandschenkestrasse 110, 8002 Zurich, Switzerland.
Identifikationsnummer: CH-020.4.028.116-1

File: step-release-vis/src/app/services/environment_test.ts (hunk 1 above)
Comment (ancamaria99) [reply step-release-vis/src/app/services/environment_test.ts]: oh, sorry. I already added for all and now I have just seen the comment after pushing :(

Comment (ancamaria99) [reply step-release-vis/src/app/services/environment_test.ts]: I'll change ASAP

**Review by stofee — APPROVED** [review]

File: step-release-vis/src/app/services/environment_test.ts (hunk 1 above)
Comment (stofee) [reply step-release-vis/src/app/services/environment_test.ts]: no need ;)

**Pull request merged**